ParcelGoApp/views.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import render
+from .models import ParcelLocker
+from .forms import ParcelLockerSearchForm
+
+
+def parcel_locker_search(request):
+    parcel_lockers = None  # Początkowo ustawiamy na None, aby nie wyświetlać listy na początku
+    form = ParcelLockerSearchForm(request.GET or None)  # Pobierz dane z formularza lub utwórz nowy formularz
+
+    if form.is_valid():
+        search_query = form.cleaned_data.get('search_query')
+        if search_query:
+            # wyszukiwanie paczkomatów po mieście lub numerze paczkomatu
+            parcel_lockers = ParcelLocker.objects.filter(
```

Comment:
from django.db.models import Q
query = Q(street__icontains=adres) | Q(city__icontains=adres)
parcel=ParcelLocker.objects.filter(query)
 To jest przykład or tutaj dokumentacja 
https://docs.djangoproject.com/en/5.0/ref/models/querysets/#or
Albo użycie Union 
https://docs.djangoproject.com/en/5.0/ref/models/querysets/#union